### ISSUE TITLE
Could you provide a simplest possible example?

### ISSUE THREAD

**Issue by MarkoPaasila** [daredoes/linked-lovelace-ui#6]
The example leaves me confused. Could you please provide an example for creating a template of the _entity_card_ card and deploying it with one single entity? No mushroom card, stack_card, or other custom cards. Thank you!

**Comment by ahbrims** [comment]
> The example leaves me confused. Could you please provide an example for creating a template of the _entity_card_ card and deploying it with one single entity? No mushroom card, stack_card, or other custom cards. Thank you!

I have the same issue. I don't know what code to write to get the templated card to display in other dashboards.  In my case I have the template `/lovelace-templates/example` with the the card: 
`type: entity
entity: sensor.time`

**Comment by stefanschaedeli** [comment]
just add a manual card with 
`template: example`

and then push the "update all " button that is show if you create another card with the following code:
`  - type: custom:linked-lovelace-ui`

**Comment by daredoes** [comment]
Hey! I'll do this soon! 

I added a new card that helps you identify existing templates you can use, as well as what template data you can pass into it. 

Although it's not documented anywhere... yet, you can pass data to a template card by surrounding anything with `$` so 
a template card of
```yaml
type: markdown-card
name: '$name$'
```
would create a parameter of 
```yaml
template: demo
template_data:
    name: 'whatever you want'
```

**Comment by daredoes** [comment]
@MarkoPaasila @ahbrims @stefanschaedeli

To make sure you aren't blocked on implementing your templates, here is a `Starter Template Dashboard` that can be dropped into your own HA instance.

https://github.com/daredoes/linked-lovelace-ui/wiki/Starter-View

**Comment by daredoes** [comment]
I should also note, the reason for the vertical stacks is because a template is limited to a `View` that has **_exactly one card_**. If there is more than one card on the view, I can't keep track of which one should be the template
